CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.9.0] - 2025-01-30
+
+### Features
+
+- Added support for [MSC3814](https://github.com/matrix-org/matrix-spec-proposals/pull/3814/),
+    allowing users to export an Olm Account in a serialized format. The
+    serialized account can be uploaded to a server for safekeeping until
+    needed.
```

Comment:
Sometimes, in this changelog, newlines in list items have no indentation, sometimes 2 spaces, and here 4 spaces. How can you live with this situation 😋?

Reply:
We'll switch to manually written changelogs here as well so this will resolve itself.